src/main/java/com/znsio/teswiz/listener/CucumberScenarioReporterListener.java

```diff
@@ -7,16 +7,16 @@
 
 import java.util.Calendar;
 
-public class CucumberWebScenarioReporterListener
+public class CucumberScenarioReporterListener
```

Comment:
We have the same class names in ATD - CucumberScenarioListener and CucumberScenairoReporterListener. We need to think of a different name for the teswiz classes to avoid confusion

Reply:
updated names